clients/privacy-center/components/modals/privacy-request-modal/PrivacyRequestForm.tsx

```diff
@@ -77,32 +70,37 @@ const usePrivacyRequestForm = ({
         return;
       }
 
-      const { email, phone, name, ...customPrivacyRequestFieldValues } = values;
+      // extract identity input values
+      const identityInputValues = Object.fromEntries(
+        Object.entries(action.identity_inputs ?? {}).map(([key, field]) => {
+          const value = values[key] || null;
```

Comment:
Do we care about handling boolean and number values here?  If `values[key]` is falsy (including `false` or `0`), this syntax will overwrite that with null.  If this is just trying to fall back if the value doesn't exist, I would prefer using `??`.

Reply:
Good catch, I added explicit checks for `undefined` and `""` before falling back to `null`.